### PR TITLE
drag and drop improvements and fs browser layout cleanup

### PR DESCRIPTION
**Pull request by songgao**

### CODE REVIEW

File: shared/fs/common/hooks.tsx [hunk 1]
```diff
@@ -54,15 +42,15 @@ const useFsNonPathSubscriptionEffect = (topic: RPCTypes.SubscriptionTopic) => {
 
 export const useFsPathMetadata = (path: Types.Path) => {
   useFsPathSubscriptionEffect(path, RPCTypes.PathSubscriptionTopic.stat)
-  const dispatch = useDispatchWhenConnectedAndOnline()
```
Comment (jakob223) [inline shared/fs/common/hooks.tsx]: why this change?

Comment (songgao) [reply shared/fs/common/hooks.tsx]: In offline mode, we still need this in synced folder. So just need to make sure KBFS is connected.
